cmd/grype/cli/options/experimental.go

```diff
@@ -1,32 +1,10 @@
 package options
 
-import (
-	"github.com/anchore/clio"
-	"github.com/anchore/grype/grype/db"
-)
-
 // Experimental options are opt-in features that are...
 // ...not stable
 // ...not yet fully supported
 // ...not necessarily tested
 // ...not ready for production use
 // these may go away at any moment, do not depend on them
 type Experimental struct {
```

Comment:
should we remove this struct altogether?

Reply:
my vote would be to keep the struct and leave it empty, that way when we have another feature flag like this it's easier to add than last time